limbo/slack.py

```diff
@@ -25,34 +25,9 @@ class SlackNotConnected(Exception): pass
 class SlackConnectionError(Exception): pass
 class SlackLoginError(Exception): pass
 
-User = namedtuple('User', 'server name id real_name tz')
+User = namedtuple('User', 'id name real_name tz')
 Bot = namedtuple('Bot', 'id name icons deleted')
-
-class Channel(object):
-    def __init__(self, server, name, id, members=[]):
-        self.server = server
-        self.name = name
-        self.id = id
-        self.members = members
-
-    def __eq__(self, compare_str):
-        if self.name == compare_str or self.name == "#" + compare_str or self.id == compare_str:
-            return True
-        else:
-            return False
-
-    def __str__(self):
-        data = ""
-        for key in list(self.__dict__.keys()):
-            data += "{0} : {1}\n".format(key, str(self.__dict__[key])[:40])
-        return data
-
-    def __repr__(self):
-        return self.__str__()
-
-    def send_message(self, message):
-        message_json = {"type": "message", "channel": self.id, "text": message}
-        self.server.send_to_websocket(message_json)
+Channel = namedtuple('Channel', 'id name')
```

Comment:
`namedtuple` was a great choice of data structure for all these entities - good idea!

it especially makes it clear when you compare this diff - so much simpler than building your own class